lua/packer.lua

```diff
@@ -439,24 +437,21 @@ local function refresh_configs(plugs)
 end
 
 local function parse_value(value)
-  if value == "true" then
-    return true
-  end
-  if value == "false" then
-    return false
-  end
+  if value == "true" then return true end
+  if value == "false" then return false end
   return value
 end
 
 local function parse_args(args)
   local result = {}
-  if not args then return result end
-  local parts = vim.split(args, ' ')
-  for _, part in ipairs(parts) do
-    if part then
-      if part:find('=') then
-        local key, value = unpack(vim.split(part, '='))
-        result[key] = parse_value(value)
+  if args then
```

Comment:
@wbthomason this change i.e. not returning an empty table will cause the line in `compile` where we check `args.profile` to error because `args` will be `nil` which was why I was returning the empty result

Reply:
I believe it should still return the empty table - if `args` is `nil`, then it skips this conditional and falls through to the `return result` on line 458, which is outside of the conditional.

Reply:
Ah I misread that 👍🏿